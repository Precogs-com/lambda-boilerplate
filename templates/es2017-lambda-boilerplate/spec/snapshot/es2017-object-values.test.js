const { transform } = require('./utils');

it('object literal', () => {
    const code = transform('Object.values({ a: 1, b: 2 });');
    expect(code).toMatchSnapshot();
});

it('object variable', () => {
    const code = transform('var obj = { a: 1, b: 2 }; Object.values(obj);');
    expect(code).toMatchSnapshot();
});
