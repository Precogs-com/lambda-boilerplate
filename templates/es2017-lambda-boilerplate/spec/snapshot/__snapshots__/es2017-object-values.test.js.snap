// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`object literal 1`] = `
"\\"use strict\\";

function _objectValues(obj) {
  var values = [];
  var keys = Object.keys(obj);

  for (var k = 0; k < keys.length; ++k) values.push(obj[keys[k]]);

  return values;
}

_objectValues({ a: 1, b: 2 });"
`;

exports[`object variable 1`] = `
"\\"use strict\\";

function _objectValues(obj) {
  var values = [];
  var keys = Object.keys(obj);

  for (var k = 0; k < keys.length; ++k) values.push(obj[keys[k]]);

  return values;
}

var obj = { a: 1, b: 2 };_objectValues(obj);"
`;
